apps/remote_control/lib/lexical/remote_control/search/store/backends/ets/schema.ex

```diff
@@ -45,7 +45,9 @@ defmodule Lexical.RemoteControl.Search.Store.Backends.Ets.Schema do
 
   alias Lexical.Project
   alias Lexical.RemoteControl.Search.Indexer.Entry
-  alias Lexical.VM.Versions
+  alias Lexical.RemoteControl.Search.Store.Backends.Ets.Wal
+
+  use Wal
```

Comment:
I don't agree with your suggestion, because it requires importing a function that you never call directly, but this works:
```elixir
import Wal, only: :macros
```
which I think is just as ergonomic.

Reply:
I think `only: :macros` is fine. However, I don't think the using module should *need* to import `operation/1` (I think that's the one that's never used directly) in order to use `with_wal`. The `with_wal` macro can do the import itself before the ets calls are rewritten. Something like:

```elixir
defmacro with_wal(wal, do: block) do
  ...

  quote do
    import unquote(__MODULE__), only: [operation: 1]
    ...
  end
end
```